_common.scss

```diff
@@ -56,4 +56,21 @@
         padding-bottom: 0.5rem;
         font-size: 0.875rem;
     }
+
+
+    .mat-button-base[blui-inline] {
+        .mat-button-wrapper {
+            display: flex;
+            align-items: center;
+        }
+        .mat-icon {
+            height: 18px;
+            width: 18px;
+            font-size: 18px;
+            margin: 0 8px 0 -4px;
```

Comment:
Vertical distance should be in `rem` units

Reply:
Hmm, for this I don't think it makes to make height in rem but width in px.  We'd want the icon to stay as a square. Should they both be in rem?

Reply:
Ah yes... that's what I meant to say. 🙈

Reply:
This is now this is updated. I'll merge this in.